spec-main/crash-spec.ts

```diff
@@ -12,9 +12,11 @@ const runFixtureAndEnsureCleanExit = (args: string[]) => {
   const child = cp.spawn(process.execPath, args);
   children.push(child);
   child.stdout.on('data', (chunk: Buffer) => {
+    console.log(chunk.toString());
```

Comment:
test apps also needs to be started with `--enable-logging` to get renderer logs which we are interested in this case,

```
const args = [fixture, '--enable-logging'];
```